# HG changeset patch
# User Massimiliano Fasi <massimiliano.fasi@manchester.ac.uk>
# Date 1523609725 -3600
#      Fri Apr 13 09:55:25 2018 +0100
# Branch stable
# Node ID 905ce390d4f5b845e560d5fedf4bb5d6d174e202
# Parent  e5341aff3cb8839b41a4beff66cd8fbd9d7c2217
Fix inaccuracies in Italian translation.

* libgui/languages/it_IT.ts: Fix typos and accelerators.
* libgui/languages/translators: Add Marco to the list of Italian translators.

diff --git a/libgui/languages/it_IT.ts b/libgui/languages/it_IT.ts
--- a/libgui/languages/it_IT.ts
+++ b/libgui/languages/it_IT.ts
@@ -234,27 +234,27 @@
     <message>
         <location filename="../qterminal/libqterminal/unix/Filter.cpp" line="+628"/>
         <source>Open Link</source>
         <translation>Apri collegamento</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Copy Link Address</source>
-        <translation>Copia l'indirizzo del collegamento</translation>
+        <translation>Copia l&apos;indirizzo del collegamento</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Send Email To...</source>
-        <translation>Manda un'email a...</translation>
+        <translation>Manda un&apos;email a...</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Copy Email Address</source>
-        <translation>Copia l'indirizzo email</translation>
+        <translation>Copia l&apos;indirizzo email</translation>
     </message>
     <message>
         <location line="+10"/>
         <location line="+12"/>
         <source>Edit %1 at line %2</source>
         <translation>Modifica %1 alla riga %2</translation>
     </message>
 </context>
@@ -768,17 +768,17 @@ in scrittura: %2.</translation>
     <message>
         <location line="+7"/>
         <source>&amp;Indent Selection Rigidly</source>
         <translation>&amp;Indenta rigidamente la selezione</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>&amp;Unindent Selection Rigidly</source>
-        <translation>&amp;Deindenta rigidamente selezione</translation>
+        <translation>D&amp;eindenta rigidamente selezione</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Indent Code</source>
         <translation>Indenta il codice</translation>
     </message>
     <message>
         <location line="+7"/>
@@ -823,32 +823,32 @@ in scrittura: %2.</translation>
     <message>
         <location line="+5"/>
         <source>Pre&amp;vious Bookmark</source>
         <translation>Segnalibro &amp;precedente</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Toggle &amp;Bookmark</source>
-        <translation>Attiva/disattiva &amp;segnalibro</translation>
+        <translation>Attiva/&amp;disattiva segnalibro</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>&amp;Remove All Bookmarks</source>
         <translation>&amp;Elimina tutti i segnalibri</translation>
     </message>
     <message>
         <location line="+8"/>
         <source>&amp;Preferences...</source>
         <translation>&amp;Preferenze...</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>&amp;Styles Preferences...</source>
-        <translation>Preferenze di &amp;stile...</translation>
+        <translation>P&amp;referenze di stile...</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>&amp;View</source>
         <translation>&amp;Vista</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -953,17 +953,17 @@ in scrittura: %2.</translation>
     <message>
         <location line="+6"/>
         <source>&amp;Help</source>
         <translation>&amp;Aiuto</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>&amp;Help on Keyword</source>
-        <translation>&amp;Aiuto su parola chiave</translation>
+        <translation>&amp;Aiuto sulla parola chiave</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>&amp;Documentation on Keyword</source>
         <translation>&amp;Documentazione sulla parola chiave</translation>
     </message>
     <message>
         <location line="+25"/>
@@ -1473,17 +1473,17 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+3"/>
         <source>Find &amp;what:</source>
         <translation>T&amp;rova:</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Re&amp;place with:</source>
-        <translation>Sos&amp;tituisci con:</translation>
+        <translation>Sostit&amp;uisci con:</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Match &amp;case</source>
         <translation>Maiuscole/&amp;minuscole</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -1493,27 +1493,27 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+1"/>
         <source>&amp;Wrap while searching</source>
         <translation>&amp;Vai a capo durante la ricerca</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Find Next</source>
-        <translation>&amp;Trova successivo</translation>
+        <translation>Trova su&amp;ccessivo</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Find &amp;Previous</source>
         <translation>Trova &amp;precedente</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>&amp;Replace</source>
-        <translation>Sos&amp;tituisci</translation>
+        <translation>&amp;Sostituisci</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Replace &amp;All</source>
         <translation>S&amp;ostituisci tutti</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -1777,17 +1777,17 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+10"/>
         <source>Filter</source>
         <translation>Filtra</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Command History</source>
-        <translation>Cronologia dei comandi</translation> <!---->
+        <translation>Cronologia dei comandi</translation>
     </message>
 </context>
 <context>
     <name>octave::initial_page</name>
     <message>
         <location filename="../src/welcome-wizard.cc" line="-178"/>
         <source>Welcome to Octave!</source>
         <translation>Benvenuto in Octave!</translation>
@@ -2013,17 +2013,17 @@ Fare click su &apos;Successivo&apos; per
     <message>
         <location line="+3"/>
         <source>Clear Command History</source>
         <translation>Svuota la cronologia dei comandi</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Clear Workspace</source>
-        <translation>Pulisci lo spazio di lavoro</translation>
+        <translation>Svuota lo spazio di lavoro</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Preferences...</source>
         <translation>Preferenze...</translation>
     </message>
     <message>
         <location line="+49"/>
@@ -2559,17 +2559,17 @@ Si desidera utilizzare comunque la scorc
         <location line="+2"/>
         <source>Clear Workspace</source>
         <translation>Pulisci lo spazio di lavoro</translation>
     </message>
     <message>
         <location line="+1"/>
         <location line="+162"/>
         <source>Preferences</source>
-        <translation>Preferenze</translation>
+        <translation>Impostazioni</translation>
     </message>
     <message>
         <location line="-159"/>
         <source>Step</source>
         <translation>Un&apos;istruzione alla volta</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -3023,17 +3023,17 @@ Si desidera utilizzare comunque la scorc
     <message>
         <location line="+2"/>
         <source>Run Selection</source>
         <translation>Esegui selezione</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Help on Keyword</source>
-        <translation>Aiuto su parola chiave</translation>
+        <translation>Aiuto sulla parola chiave</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Document on Keyword</source>
         <translation>Documentazione su parola chiave</translation>
     </message>
     <message>
         <location line="+57"/>
@@ -3536,23 +3536,23 @@ Vorresti salvare le attuali scorciatoie 
         <location filename="../src/ui-settings-dialog.h" line="+1"/>
         <source>Graphic icons</source>
         <translation>Icone grafiche</translation>
     </message>
     <message>
         <location line="-29"/>
         <location filename="../src/ui-settings-dialog.h" line="-3"/>
         <source>Dock widget title bar</source>
-        <translation>Barra del titolo del lanciatore di widget</translation>
+        <translation>Barra del titolo degli widget</translation>
     </message>
     <message>
         <location line="-176"/>
         <location filename="../src/ui-settings-dialog.h" line="-8"/>
         <source>Confirm before exiting</source>
-        <translation>Conferma prima di uscire</translation>
+        <translation>Chiedi conferma prima di uscire</translation>
     </message>
     <message>
         <location line="+95"/>
         <location filename="../src/ui-settings-dialog.h" line="+5"/>
         <source>Custom style</source>
         <translation>Stile personalizzato</translation>
     </message>
     <message>
@@ -3650,17 +3650,17 @@ Vorresti salvare le attuali scorciatoie 
         <location filename="../src/ui-settings-dialog.h" line="-66"/>
         <source>Show horizontal scroll bar</source>
         <translation>Mostra barra di scorrimento orizzontale</translation>
     </message>
     <message>
         <location line="+431"/>
         <location filename="../src/ui-settings-dialog.h" line="+25"/>
         <source>Wrap long lines at current window border</source>
-        <translation>Spezza righe lunghe al marigne della finsetra corrente</translation>
+        <translation>Spezza righe lunghe al margine della finestra corrente</translation>
     </message>
     <message>
         <location line="+55"/>
         <location filename="../src/ui-settings-dialog.h" line="+2"/>
         <source>Indent width</source>
         <translation>Ampiezza indentazione</translation>
     </message>
     <message>
@@ -4218,17 +4218,17 @@ Vorresti salvare le attuali scorciatoie 
         <location filename="../src/ui-settings-dialog.h" line="+1"/>
         <source>Autofit</source>
         <translation>Adatta automaticamente</translation>
     </message>
     <message>
         <location line="-63"/>
         <location filename="../src/ui-settings-dialog.h" line="-5"/>
         <source>Plus font height</source>
-        <translation>Più altezza del carattere</translation>
+        <translation>Oltre all&apos;altezza del carattere</translation>
     </message>
     <message>
         <location line="+77"/>
         <location filename="../src/ui-settings-dialog.h" line="+8"/>
         <source>By Column</source>
         <translation>Per colonna</translation>
     </message>
     <message>
@@ -4254,17 +4254,17 @@ Vorresti salvare le attuali scorciatoie 
         <location filename="../src/ui-settings-dialog.h" line="-22"/>
         <source>HttpProxy</source>
         <translation>HttpProxy</translation>
     </message>
     <message>
         <location line="-2179"/>
         <location filename="../src/ui-settings-dialog.h" line="-139"/>
         <source>Icon set for dock widgets</source>
-        <translation>Icone per il lanciatore di widget</translation>
+        <translation>Set di icone per gli widget</translation>
     </message>
     <message>
         <location line="-220"/>
         <location filename="../src/ui-settings-dialog.h" line="-11"/>
         <source>Language (requires restart)</source>
         <translation>Lingua (cambiarla richiede riavvio)</translation>
     </message>
     <message>
diff --git a/libgui/languages/translators b/libgui/languages/translators
--- a/libgui/languages/translators
+++ b/libgui/languages/translators
@@ -2,16 +2,16 @@
 # to be able to inform them about generic translation file changes.
 # The generic file that can be used as a template for new
 # translation files is en_US.ts
 be_BY Mihas Varantsou <meequz@gmail.com>
 de_DE Andreas Weber <andy.weber.aw@googlemail.com>
 es_ES Valentin Ortega-Clavero <arcanos3030@gmail.com>, José Luis García Pallero <jgpallero@gmail.com>
 eu_ES Ander Aurrekoetxea <zaurreko@gmail.com>
 fr_FR David Bateman <dbateman@free.fr>, Catalin Codreanu <codreanu.catalin@gmail.com>
-it_IT Massimiliano Fasi <massimiliano.fasi@gmail.com>
+it_IT Massimiliano Fasi <massimiliano.fasi@gmail.com>, Marco Caliari <marco.caliari@univr.it>
 ja_JP Tatsuro Matsuoka <tmacchant@yahoo.co.jp>
 pt_BR Júlio Hoffimann Mendes <julio.hoffimann@gmail.com>, Felipe G. Nievinski <fgnievinski@gmail.com>
 pt_PT Carnë Draug <carandraug@octave.org>
 ru_RU Andriy Shinkarchuck <adriano32.gnu@gmail.com>, Dmitry Roshchin <dmitry@roshchin.org>
 uk_UA Andriy Shinkarchuck <adriano32.gnu@gmail.com>
 nl_NL Sander van Rijn <svr003@gmail.com>
 zh_CN Jeff Bai <aosc@members.fsf.org>
